crates/fonticulus/src/fontinfo.rs

```diff
@@ -131,3 +132,17 @@ pub fn get_panose(_info: &norad::FontInfo) -> fonttools::os2::Panose {
         panose9: 0,
     }
 }
+pub fn get_selection(info: &norad::FontInfo) -> u16 {
+    let selection = int_list_to_num(&info.open_type_os2_type.as_ref()
+        .unwrap_or(&vec![0])) as u16;
+    let style_map = style_map_style_name(info);
```

Comment:
style_map_style_name is more sophisticated in ufo2ft. If the ufo hasn't specified this attrib, ufo2ft will first try and assemble it from the prefferedSubFamily name and lastly try the subFamily. I will implement this functionality.